sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -1,19 +1,19 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 12.0.0-beta.5 (2020-11-13)
 
 ### Breaking Changes
 
-* Move header `azure/storage/files/datalake/datalake.hpp` to `azure/storage/files/datalake.hpp`
+* Moved header `azure/storage/files/datalake/datalake.hpp` to `azure/storage/files/datalake.hpp`
+* Moved returning model types and related functions in `Azure::Storage::Files::DataLake` to `Azure::Storage::Files::DataLake::Models`, and other code into `Azure::Storage::Files::DataLake::Details`.
```

Comment:
Phrasing it to highlight the thing that would impact/matter to the user would be useful. Why is moving things to ::Details important? Because now they are private and the user shouldn't call them directly, especially since they can change:
```suggestion
* Moved returning model types and related functions in `Azure::Storage::Files::DataLake` to `Azure::Storage::Files::DataLake::Models`, and made other code private by moving it into `Azure::Storage::Files::DataLake::Details`.
```

Reply:
It might be better to have consistent phrasing here, matching what's in: https://github.com/Azure/azure-sdk-for-cpp/pull/974